app/views/posts/index.html.erb

```diff
@@ -1 +1,27 @@
-<h1>Hello new Post</h1>
\ No newline at end of file
+<div id="posts_container">
+  <div id="user_container">
+  <img id="user_img" src="<%= @user.photo %>" alt="user profile">
+      <div id="container_container_div">
+        <h3><%= @user.name %></h3>
+        <p>Number of posts: <%= @user.posts_counter %></p>
+      </div>
+    </div>
+    <% @posts[0...5].each do |post|%>
+    <div id="user_posts">
```

Comment:
- [ ] The range loop prevents the display of all the posts using the button `See all posts`.

![all-posts](https://user-images.githubusercontent.com/53124548/216294563-10a10829-4322-4bb6-9688-25ea0ed13bf2.png)